ext/bcmath/bcmath.c

```diff
@@ -179,9 +177,9 @@ PHP_FUNCTION(bcadd)
 		goto cleanup;
 	}
 
-	bc_add (first, second, &result, scale);
+	result = bc_add (first, second, scale);
```

Comment:
```suggestion
	bc_num result = bc_add (first, second, scale);
```

Maybe can declare and initialize it at the same time and remove the result from the cleanup.

But I'm not sure if it affects performance.

Reply:
Good question. I expect this will only make the failure path potentially faster, as it can avoid a call then to the freeing. For the success path I don't think it matters, but I guess feel free to try.